### PR TITLE
[UIPO-43] Product list component

### PR DESCRIPTION
**Pull request by mirafedas**

### CODE REVIEW

File: src/components/Brands/Brands.js [hunk 1]
```diff
@@ -46,8 +45,8 @@ class BrandsAndShippingInfo extends Component {
     const { brandsList } = this.state;
 
     return (
-      <div className={cx('content', CN)}>
-        <Heading title="Brands" position="center" />
+      <div className="content">
+        <h3 className="title">Top Brands</h3>
```
Comment (Nazar-bnm) [inline src/components/Brands/Brands.js]: Please use bem here

File: src/components/BurgerMenu/BurgerMenu.scss [hunk 2]
```diff
@@ -1,43 +1,49 @@
...
-  padding-right: 10px;
-
   @keyframes open {
     from {
-      left: -80vw;
+        left: -100vw;
     }
     to {
-      left: 0;
+        left: 0;
```
Comment (Nazar-bnm) [inline src/components/BurgerMenu/BurgerMenu.scss]: remove extra tabs

File: src/components/BurgerMenu/BurgerMenu.scss [hunk 3]
```diff
@@ -59,13 +65,13 @@
...
     display: block;
-    width: 120px;
+    width: auto;
   }
 
   &__link {
     font-size: 0.9rem;
     text-decoration: none;
-    background-color: $white;
+    background-color: #ffffff;
```
Comment (Nazar-bnm) [inline src/components/BurgerMenu/BurgerMenu.scss]: why did you change it ?

File: src/components/Header/PageSettings/PageSettings.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ const PageSettings = () => (
   <div className={cx(CN, 'content container')}>
     <ContentOptions />
     <h3 className={`${CN}__heading heading`}>
-      free shipping on orders above 50$
+        free shipping on orders above 50$
```
Comment (Nazar-bnm) [inline src/components/Header/PageSettings/PageSettings.js]: ???

File: src/components/MostPopular/ProductItem/ProductItem.js [hunk 5]
```diff
@@ -1,34 +1,92 @@
...
+      message: 'The item added to the wishlist',
+      type: 'success'
+    }
+  };
+
+  const audio = new Audio('src/assets/sounds/notification-sound.mp3');
+
+  const playAudio = () => {
+    audio.play();
+  };
```
Comment (Nazar-bnm) [inline src/components/MostPopular/ProductItem/ProductItem.js]: why do you use Audio here ? It should be in Notification Component

File: src/components/ProductList/ItemsList/ItemsList.js [hunk 6]
```diff
@@ -49,29 +71,53 @@ class ItemsList extends React.Component {
...
+        addToWishList={addToWishList}
+        removeFromWishList={removeFromWishList}
+      />
     ));
   }
 
+  getCategoriesURL() {
+    const {
+      filters
+    } = this.props;
```
Comment (Nazar-bnm) [inline src/components/ProductList/ItemsList/ItemsList.js]: write it in one line

File: src/components/ProductList/ItemsList/ItemsList.js [hunk 7]
```diff
@@ -49,29 +71,53 @@ class ItemsList extends React.Component {
...
+    sizes.forEach((item) => {
+      categories += `&sizes=${item}`;
+    });
+    colors.forEach((item) => {
+      categories += `&colors=${item}`;
+    });
+    brands.forEach((item) => {
+      item.replace(regexWhiteSpace, '');
+      categories += `&brandId=${brandIDEnum[item]}`;
+    });
```
Comment (Nazar-bnm) [inline src/components/ProductList/ItemsList/ItemsList.js]: please change it to use only one itteration

File: src/components/ProductList/ItemsList/ItemsList.js [hunk 8]
```diff
@@ -84,15 +130,43 @@ class ItemsList extends React.Component {
...
 
     return (ascendingOrder ? prevEl.price - nextEl.price : nextEl.price - prevEl.price);
   }
 
+  sortItems() {
+    const {
+      ascendingOrder,
+      orderType,
+      itemList
+    } = this.props;
```
Comment (Nazar-bnm) [inline src/components/ProductList/ItemsList/ItemsList.js]: write in one line

File: src/components/WhatIsNew/ItemInfo/ItemInfo.scss [hunk 9]
```diff
@@ -81,11 +76,12 @@
       }
     }
   }
-
+
```
Comment (Nazar-bnm) [inline src/components/WhatIsNew/ItemInfo/ItemInfo.scss:79]: Why ?

**Review by Nazar-bnm — CHANGES_REQUESTED** [review]

**Comment by Nazar-bnm** [comment]
Please don't make such big and long code changes in one PR

File: src/components/MostPopular/ProductItem/ProductItem.js (hunk 5 above)
Comment (mirafedas) [reply src/components/MostPopular/ProductItem/ProductItem.js]: This change has been added by Dima Hnatkivsky, I only pulled it from the development when I was updating my branch. I will consult with him and we will make proper changes on his branch.

**Review by Nazar-bnm — APPROVED** [review]
